qt-models/divetripmodel.cpp

```diff
@@ -1427,6 +1428,24 @@ void DiveTripModelTree::divesSelectedTrip(dive_trip *trip, const QVector<dive *>
 	}
 }
 
+void DiveTripModelTree::tripSelected(dive_trip *trip, dive *currentDive)
+{
+	if (!trip)
+		return;
+
+	// Find the trip.
+	int idx = findTripIdx(trip);
+	if (idx < 0) {
+		// We don't know the trip - this shouldn't happen. We seem to have
+		// missed some signals!
+		qWarning() << "DiveTripModelTree::tripSelected(): unknown trip";
```

Comment:
would it make sense to at least spew out the trip date so there's something we can do with the warning message?